build/depends.py

```diff
@@ -310,6 +308,10 @@ def configure(self, build, conf):
             elif not qt5 and not conf.CheckForPKG('QtCore', '4.6'):
                 raise Exception('QT >= 4.6 not found')
 
+            if not conf.CheckLib('Qt5X11Extras'):
+                raise Exception('Could not find Qt5X11Extras or its development headers')
+            build.env.Append(LIBS = 'Qt5X11Extras')
```

Comment:
I think this is redundant.

Reply:
You're right. It works when this line is removed.